extensions/ql-vscode/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Prints end-of-query evaluator log summaries to the Query Log. [#1349](https://github.com/github/vscode-codeql/pull/1349)
 - Be consistent about casing in Query History menu. [#1369](https://github.com/github/vscode-codeql/pull/1369)
+- Use the same Node version as VSCode [#1374](https://github.com/github/vscode-codeql/pull/1374)
```

Comment:
I don't think this requires a changelog post since this only affects the dev experience.